src/Builder/FrontendDialectHelper.cpp

```diff
@@ -126,8 +126,9 @@ mlir::Value InitializedTensorMapping::EmitInitializerForInputTensor(
       onnxTensorProtoToDenseElmAttr(builder, initializer);
 
   // Create ConstantOp for dense array.
-  return builder.create<mlir::ONNXConstantOp>(
-      loc, denseElmAttr.getType(), nullptr, denseElmAttr);
+  return builder.create<mlir::ONNXConstantOp>(loc, denseElmAttr.getType(),
```

Comment:
@chentong319 would there be a way to add a custom constructor with fewer parameters, where all the nullptr parameters below are set to null inside the constructor? I think there are example of custom builder already.

Reply:
Yes, I wrote that constructor and forgot to use it here.